docs/user/ppl/functions/datetime.rst

```diff
@@ -38,8 +38,38 @@ Example::
     | 2020-08-26 01:00:00                            | 2020-08-27                       | 2020-08-27 01:01:01                            |
     +------------------------------------------------+----------------------------------+------------------------------------------------+
 
+
+ADDTIME
+-------
+
+Description
+>>>>>>>>>>>
+
+Usage: addtime(expr1, expr2) adds expr2 to expr1 and returns the result. If argument is TIME, today's date is used; if argument is DATE, time at midnight is used.
+
+Argument type: DATE/DATETIME/TIMESTAMP/TIME, DATE/DATETIME/TIMESTAMP/TIME
+
+Return type map:
+
+(DATE/DATETIME/TIMESTAMP, DATE/DATETIME/TIMESTAMP/TIME) -> DATETIME
+
+(TIME, DATE/DATETIME/TIMESTAMP/TIME) -> TIME
+
+Antonyms: `SUBTIME`_
+
+Example::
+
+    os> source=people | eval `'2008-12-12' + 0` = ADDTIME(DATE('2008-12-12'), DATE('2008-11-15')), `'23:59:59' + 0` = ADDTIME(TIME('23:59:59'), DATE('2004-01-01')), `'2004-01-01' + '23:59:59'` = ADDTIME(DATE('2004-01-01'), TIME('23:59:59')), `'10:20:30' + '00:05:42'` = ADDTIME(TIME('10:20:30'), TIME('00:05:42')) | fields `'2008-12-12' + 0`, `'23:59:59' + 0`, `'2004-01-01' + '23:59:59'`, `'10:20:30' + '00:05:42'`
+    fetched rows / total rows = 1/1
+    +---------------------+------------------+-----------------------------+---------------------------+
+    | '2008-12-12' + 0    | '23:59:59' + 0   | '2004-01-01' + '23:59:59'   | '10:20:30' + '00:05:42'   |
+    |---------------------+------------------+-----------------------------+---------------------------|
+    | 2008-12-12 00:00:00 | 23:59:59         | 2004-01-01 23:59:59         | 10:26:12                  |
+    +---------------------+------------------+-----------------------------+---------------------------+
```

Comment:
I manually tested it and couldn't find any issues, but I would add an example of ADDTIME where the day switches because of the time addition.
Something like below:
```
opensearchsql> SELECT ADDTIME(DATETIME('2008-02-29 23:50:00'), TIME('10:10:10')); 
..............                                                                                                                                                                                                                                              
fetched rows / total rows = 1/1
+--------------------------------------------------------------+
| ADDTIME(DATETIME('2008-02-29 23:50:00'), TIME('10:10:10'))   |
|--------------------------------------------------------------|
| 2008-03-01 10:00:10                                          |
+--------------------------------------------------------------+
```

Reply:
Fixed in db18570.